<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_plot</class>
 <widget class="QWidget" name="Form_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,3">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_left">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4,4">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_Tittle">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>EtherCAT采集应用-八通道AD采集</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="widget_plotStandard" native="true"/>
        </item>
        <item>
         <widget class="QCustomPlot" name="widget_plotFFT" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_right">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>199</width>
          <height>477</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_Parameter">
           <property name="title">
            <string>Parameter</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Slave Index</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_SlaveIndex"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>OS Channel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_SampleOS_CH">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(80, 80, 80);</string>
                </property>
                <item>
                 <property name="text">
                  <string>OS_NONE</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>OS_CH1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>OS_CH1_CH2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>OS_CH1_CH4</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>OS_CH1_CH8</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>AD Channel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_AD_CH">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(80, 80, 80);</string>
                </property>
                <item>
                 <property name="text">
                  <string>AD_CH1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AD_CH2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AD_CH3</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AD_CH4</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AD_CH5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AD_CH6</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AD_CH7</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AD_CH8</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Sample Rate (kHz)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Sample_Rate">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(80, 80, 80);</string>
                </property>
                <property name="text">
                 <string>20</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QDial" name="dial_SampleRate"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_Buttons">
           <property name="title">
            <string>Buttons</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_PlotStart">
              <property name="text">
               <string>开始采集</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_PlotStop">
              <property name="text">
               <string>停止采集</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_DisPlayReset">
              <property name="text">
               <string>显示复位</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_Dispaly">
           <property name="title">
            <string>Display</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Display Num</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_DisplayNum">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(80, 80, 80);</string>
                </property>
                <property name="text">
                 <string>1024</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QDial" name="dial_Display_Num"/>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>State</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_ErrState">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(80, 80, 80);</string>
                </property>
                <property name="text">
                 <string>Error_None</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
